doc/intro.xml

```diff
@@ -18,6 +18,11 @@ in this package is useful at least for permutation groups, matrix
 groups and projective groups.
 The setup is described in <Cite Key="NS06"/>. <P/>
 
+<Cite Key="LNPS06"/>
+<Cite Key="BLS97"/>
+<Cite Key="JLNP13"/>
+<Cite Key="KS09"/>
```

Comment:
These are currently just here so that they appear in the bibliography in the compiled doc.

Reply:
Makes sense